test/controllers/managing.officer.controller.spec.ts

```diff
@@ -7,7 +7,7 @@ jest.mock('../../src/utils/save.and.continue');
 jest.mock("../../src/utils/logger");
 jest.mock('../../src/middleware/navigation/has.beneficial.owners.statement.middleware');
 
-import { describe, expect, test, jest } from '@jest/globals';
+import { describe, expect, test, jest, beforeEach } from '@jest/globals';
```

Comment:
is the beforeEach needed? i couldn't see it used in the changes

Reply:
It is needed on line 89, it was missing, I just added to fix ts error on my IDE